src/main/scala/edu/ie3/simona/agent/participant/evcs/EvcsAgentFundamentals.scala

```diff
@@ -442,19 +443,20 @@ protected trait EvcsAgentFundamentals
         baseStateData,
       )
 
-    val stayingSchedules =
-      lastState.schedule
-        .filterNot(requestedDepartingEvs.contains)
-        .view
-        .mapValues {
-          // Remove schedules that ended before or at current tick.
-          // Schedule entries ending at current tick do not have any
-          // impact on the schedule from the current tick on
-          _.filter(_.tickStop > tick)
-        }
-        .toMap
+    // filter(_.tickStop > currentTick)
+    // TODO is it possible to remove also the schedules that ended at currentTick? -> probably yes, test required
```

Comment:
Should be possible because we're done with these events at this point, right?

Reply:
This is no longer valid with the rollback above, i guess.